lib/cc/engine/fingerprint.rb

```diff
@@ -33,7 +33,13 @@ def compute
       attr_reader :path, :cop_name, :message
 
       def stripped_message
-        message.gsub(/ \[.+\]$/, "")
+        # example full message with details:
+        # Metrics/MethodLength: Method has too many lines. [18/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)
+        url_regex = %r{ \(https\://github\.com.+\)}
```

Comment:
Not always - the url can be overridden with the StyleGuide and Reference keys: https://github.com/bbatsov/rubocop/blob/da01214a7de225dd62ebe2df9ae54d56d7a43e54/lib/rubocop/cop/cop.rb#L226:L234